### PR TITLE
Remove all remaining references to "DPMS"

### PR DESCRIPTION
**Pull request by emersion**
Follow-up for https://github.com/swaywm/sway/pull/7083

### CODE REVIEW

File: include/sway/config.h [hunk 1]
```diff
@@ -273,6 +267,7 @@ enum render_bit_depth {
 struct output_config {
 	char *name;
 	int enabled;
+	int power;
```
Comment (bl4ckb0ne) [inline include/sway/config.h:270]: Why not make `power` a `enum zwlr_output_power_v1_mode`?

File: sway/desktop/output.c [hunk 2]
```diff
@@ -1028,10 +1028,10 @@ void handle_output_power_manager_set_mode(struct wl_listener *listener,
 	struct output_config *oc = new_output_config(output->wlr_output->name);
 	switch (event->mode) {
```
Comment (bl4ckb0ne) [inline sway/desktop/output.c:1029]: Could be simplified in `oc->power = event->mode` but it'll lose some readability.

**Comment by tiosgz** [comment]
`grep` (after filtering out obvious false positives) still reports these, have you left them there intentionally?

```
config.in:40:#          timeout 600 'swaymsg "output * dpms off"' resume 'swaymsg "output * dpms on"' \
include/sway/output.h:35:       // last applied mode when the output is DPMS'ed
sway/desktop/output.c:766:              // We mark the output enabled even if it is switched off by DPMS
```

File: include/sway/config.h (hunk 1 above)
Comment (emersion) [reply include/sway/config.h:270]: 1. I don't want to tie this to a particular Wayland protocol
2. Consistency with the rest of the fields
3. Doesn't have a "default"/"unset" enum entry

File: sway/desktop/output.c (hunk 2 above)
Comment (emersion) [reply sway/desktop/output.c:1029]: I'd rather not rely on the values meaning the same in the protocol and in the Sway codebase.

**Comment by emersion** [comment]
Thanks, fixed!

**Review by vyivel — APPROVED** [review]

**Comment by vyivel** [comment]
Thanks!

**Pull request merged**